cylc/flow/scripts/remote_init.py

```diff
@@ -14,7 +14,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-"""cylc remote-init [--indirect-comm=ssh] UUID RUND
+"""cylc remote-init [--indirect-comm=ssh] INSTALL_TARGET RUND
```

Comment:
Is `[DIRS_TO_BE_SYMLINKED ...]` meant to be here too?

Reply:
Yes, just redoing the docstrings for this now. I'll get them tidied before merge. Thanks!

Reply:
Ooh, well spotted.